dashboard/app/views/levels/_admin.html.haml

```diff
@@ -164,7 +164,7 @@
           Bookmark as a featured project
 
       #unpublished_warning{'style' => 'display: none; margin-top: 20px; width: 250px;'}
-        * This project is currently unpublished. It can still be marked as featured, but it will not show in the gallery until the owner publishes it.
+        * This project is currently unpublished. It can still be bookmarked, but it will not show in the featured projects gallery until it is published.
```

Comment:
should this say "submitted"?

Reply:
Good question! I was going to update this to "submitted", but this is the extra links box for legacy lab projects. Users can't submit legacy lab projects yet, and in order for a legacy project to be displayed in the featured gallery, an engineer has manually 'publish' the project, i.e., call on the [`publish` method in `projects.rb`](https://github.com/code-dot-org/code-dot-org/blob/88786277046df19d3728b556b3619d2e6c8f0321/dashboard/legacy/middleware/helpers/projects.rb#L127).